main.py

```diff
@@ -3,13 +3,14 @@
 from environment import CarRacingEnv
 from trainer import Trainer
 
-# if gpu is to be used
+# if gpu is to be used. unfortunately my linux doesn't have drivers for that.
 device = torch.device("cuda") if False else torch.device("cpu")
 
+# Main loop
```

Comment:
this is a bullshit comment, who the fuck put it in there.

Reply:
no, tio. que esto lo peta. creo que si lo miras de otra manera seras mas feliz.